### ISSUE TITLE
Non-recoverable issue

### ISSUE THREAD

**Issue by Kriss8089** [raiguard/RecipeBook#159]
Recipe book threw an error when trying to load one of my saves, even though it worked fine earlier in the day. I have no idea what it means, so here it is:

![Screenshot (212)](https://github.com/raiguard/RecipeBook/assets/71360533/7535ebf8-d553-4928-863d-7a56222cdf9e)

**Comment by raiguard** [comment]
Did you change mod contents without changing that mod's version number?

**Comment by Kriss8089** [comment]
That is most likely what happened. I had manually tweaked a mod to add in a few things for personal use, which is probably why I only got that error when trying to load a save created before my edits with that mod installed, but not with any other. Thanks for this knowledge though, I will keep it in mind whenever I decide to deconstruct code again.

This error didn't have any consequences though, none of my other games had problems and I had about ten minutes in the save that did crash.
